### PR TITLE
Book Detail VI - Redirection

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Completed book redirection to their detail
- **General PR fixes**

## Screenshot
Captura de Pantalla 2020-11-03 a la(s) 11 41 29 a  m

## Trello Card
https://trello.com/c/5zqugefM/33-detalle-de-libro-vi-redirecci%C3%B3n

### CODE REVIEW

File: maquetacion/src/scss/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@mixin container($width) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_mixins.scss]: Sort by name

File: maquetacion/src/scss/_styles.scss [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+  
+  &__link {
+    
+    font-size: 20px;
+    height: 20px;
+    line-height: 24px;
+    width: 91px;
+    position: relative;
+
+    // width: 59px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: remove comment

File: maquetacion/src/scss/_styles.scss [hunk 3]
```diff
@@ -1,23 +1,55 @@
...
 
-.text-light {
-  color: $grey-light;
+.back-button {
+
+  margin: 50px 0 110px 0;
+  width: 100%;
+  
+  &__link {
+
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: remove empty line

File: maquetacion/src/scss/_styles.scss [hunk 4]
```diff
@@ -1,23 +1,55 @@
...
-body {
-  font-size: 16px;
+html, body {
+  height: 100%;
 }
 
-.text-light {
-  color: $grey-light;
+.back-button {
+
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: Remove line

File: maquetacion/src/scss/_styles.scss [hunk 5]
```diff
@@ -1,23 +1,55 @@
...
+    position: relative;
+
+    // width: 59px;
+
+    &::before {
+      content: url('../../assets/back-arrow-20.png');
+      padding: 0 10px 0 0;
+      position: relative;
+      top: 2.5px;
+      bottom: 0;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: order by name

Comment (idcuadrowolox) [reply maquetacion/src/scss/_styles.scss]: it's bottom or top

File: maquetacion/src/scss/_styles.scss [hunk 6]
```diff
@@ -1,23 +1,55 @@
...
+    height: 20px;
+    line-height: 24px;
+    width: 91px;
+    position: relative;
+
+    // width: 59px;
+
+    &::before {
+      content: url('../../assets/back-arrow-20.png');
+      padding: 0 10px 0 0;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: use padding right

File: maquetacion/src/scss/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,11 @@
+// COLORS
+
+$grey-light: #828282;
+$wolox-blue: #00ADEE;
+$wolox-green: #BED23C;
+$half-black: rgba(0,0,0,0.5);
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_variables.scss]: create file for this and order by name

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_navbar.scss [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    align-items: center;
+    display: flex;
+    height: 70px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__logo {
+    height: 70px;
+    width: 212.42px;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_navbar.scss]: We usually don't use decimals for sizes since it makes no difference. Just `212px` should be fine 😄

**Review by mcallegari10 — CHANGES_REQUESTED** [review]
Great job! 🎉 Check those comments 😄

File: maquetacion/src/scss/_mixins.scss (hunk 1 above)
Comment (seromarin) [reply maquetacion/src/scss/_mixins.scss]: Sorted

**Review by seromarin — COMMENTED** [review]
All comments resolved

File: maquetacion/src/scss/_styles.scss (hunk 4 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Removed

File: maquetacion/src/scss/_styles.scss (hunk 3 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Removed

**Review by seromarin — COMMENTED** [review]
Some conflicts resolved

File: maquetacion/src/scss/_navbar.scss (hunk 8 above)
Comment (seromarin) [reply maquetacion/src/scss/_navbar.scss]: Yep, I changed to 215px

File: maquetacion/src/scss/_variables.scss (hunk 7 above)
Comment (seromarin) [reply maquetacion/src/scss/_variables.scss]: Done

File: maquetacion/src/scss/_styles.scss (hunk 6 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Done

File: maquetacion/src/scss/_styles.scss (hunk 5 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Sorted and bottom removed

File: maquetacion/src/scss/_styles.scss (hunk 4 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Done

**Review by idcuadrowolox — APPROVED** [review]

File: maquetacion/src/scss/_styles.scss (hunk 2 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Removed

**Pull request merged**